ui/pages/confirm-signature-request/index.js

```diff
@@ -74,6 +80,52 @@ const ConfirmTxScreen = ({ match }) => {
   const [prevValue, setPrevValues] = useState();
   const history = useHistory();
 
+  const getTxData = useCallback(() => {
```

Comment:
Why was this moved?

Reply:
`txData` is not defined otherwise

Reply:
I don't understand. Couldn't we just add the insight hook after where txData was defined before?

Reply:
lol yeah woops